actions.yml

```diff
@@ -1,15 +1,17 @@
 name: "npm audit"
-description: "Runs npm audit for the production dependencies of the uk-mobile repo"
+description: "Runs audit checks of npm audit and owasp-zap reports"
 inputs:
-    AUDIT_COMMAND:
-        description: "The audit command to run in a given repo"
+    REPORT_INPUT:
+        description: "The path where report is located"
```

Comment:
Maybe some more information should be added here, e.g. "[...] where the audit report [...]".

Reply:
The text has been updated.